qemu/tests/cfg/nmi_bsod_catch.cfg

```diff
@@ -19,8 +19,23 @@
     config_cmd6 = reg add "HKEY_CURRENT_USER\Software\Microsoft\Windows\Windows Error Reporting" /v Disabled /d 1 /t REG_DWORD /f
     manual_reboot_cmd =
     reboot_after_config = yes
+    i386:
+        windbg_path = "x86\windbg.exe"
+        devcon_dirname += "x86"
+    x86_64:
+        windbg_path = "x64\windbg.exe"
+        devcon_dirname += "amd64"
+    chk_windbg_cmd = 'dir "C:\Program Files (x86)\Windows Kits\10\Debuggers\%s"'
```

Comment:
The `Windows Kits\10\` is available for other Windows OS?
Since I don't know if the number `10` relates to the OS version.